packit/config/aliases.py

```diff
@@ -5,13 +5,25 @@
 from datetime import timedelta
 
 from cachetools.func import ttl_cache
+from opensuse_distro_aliases import get_distro_aliases
 
 from packit.exceptions import PackitException
 from packit.utils.bodhi import get_bodhi_client
 from packit.utils.commands import run_command
 from packit.utils.decorators import fallback_return_value
 from packit.utils.monitoring import Pushgateway
 
+OPENSUSE_ALIASES = get_distro_aliases()
```

Comment:
We can't do this at the top level, because this function gets called on module import. The problem with this is, that now importing fails if you have no internet connection or if the buildservice is down. Instead this needs to be integrated in `packit/config/aliases.py`

Reply:
A few ideas that might help here:

* Hide in a method and use a cache decorator to run it only once on the first call.
* Provide hardcoded values as a backup in case of network/system is down.
  * On the other hand, both situations are serious and the build won't be possible nevertheless -- so, a retry to overcome flakes and a nice error message if the retry won't help is sufficient I think.